### PR TITLE
standardize buttons

### PR DESCRIPTION
**Pull request by orangejenny**
there's no logic to how we use `.btn-inverse`

@nickpell / @emord

### CODE REVIEW

File: src/templates/multimedia_queue.html [hunk 1]
```diff
@@ -12,9 +12,6 @@
         <p class="text-error hqm-error hide">
             <span class="label label-important">There were errors.</span>
         </p>
-        <a href="#" class="btn btn-inverse btn-mini pull-right hqm-remove">
```
Comment (emord) [inline src/templates/multimedia_queue.html]: Why did you delete this button?

Other than this :+1: to merge

Comment (orangejenny) [inline src/templates/multimedia_queue.html]: Seemed clutter-y in a rather small dialog - you can already select a new image to overwrite the first one, or close the dialog altogether.
